src/main/res/values/preferences.xml

```diff
@@ -31,13 +31,18 @@
     <string name="preference_title_account_sign_out">Sign out</string>
     <string name="preference_title_server_connection">Connection</string>
     <string name="preference_title_tell_a_friend">Tell a friend</string>
-    <string name="preference_title_upload_settings">Auto Upload</string>
+    <string name="preference_title_upload_settings">Auto Upload Photos</string>
 
     <string name="preference_title_upload_hda">HDA Server</string>
+    <string name="preference_title_upload_server">Server</string>
     <string name="preference_title_upload_share">Share</string>
     <string name="preference_title_upload_path">Path</string>
```

Comment:
Change this title to `Location` instead of `Path` as pointed out in the comments above.

Reply:
ok, thanks. I will change it.